float64[] ArmJointPositions
float64[] ArmJointVelocities
float64[] BasePositions
float64[] BaseOrientations
float64[] BaseLinVelocities
float64[] BaseRotVelocities

duration time_from_start
